### ISSUE TITLE
Feature request: Add all

### ISSUE THREAD

**Issue by pdworkman** [kevboh/longform#133]
After setting up a dozen new projects, with 3-4 drafts each, with up to 60 scenes in each, my RSI is getting to me. It would be nice if there was an option to "add all" scenes instead of having to click add for each...

**Comment by radioactive-penguin** [comment]
As someone coming from Scrivener to Obsidian/Longform, this would be a handy feature to have.

**Comment by kevboh** [comment]
Yep, makes a lot of sense and pretty easy to do. I need to do a pass in which I use the new Obsidian metadata API, this would go well with that.
